### PR TITLE
Display and filter users by organisations

### PR DESCRIPTION
**Pull request by danielroseman**
Add a column for organisation in the users index, and allow filtering by org.

### CODE REVIEW

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -28,12 +29,14 @@
   <tbody>
     <% @users.each do |user| %>
       <tr>
-        <td class="email">
+        <td class="email" nowrap>
```
Comment (fofr) [inline app/views/admin/users/index.html.erb]: The `nowrap` attribute on a td element is obsolete. This should be achieved using CSS.

**Comment by fofr** [comment]
I like this feature.

I think the organisation dropdown should show an autocomplete select2 box when opened, to avoid a long list. The items in the list should also contain the abbreviations to make searching easier, eg DEFRA, MOJ.

screen shot 2015-01-06 at 18 33 34

**Comment by fofr** [comment]
The selected filter value might benefit from showing the shortened organisation name if available:

screen shot 2015-01-06 at 18 40 13

For example:

screen shot 2015-01-06 at 18 42 22

File: app/views/admin/users/index.html.erb [hunk 2]
```diff
@@ -28,12 +29,14 @@
...
           <%= user.suspended? ? "<del>".html_safe : "" %>
-            <%= link_to "#{user.name} <#{user.email}>", edit_admin_user_path(user) %>
+            <%= link_to "#{user.name}", edit_admin_user_path(user) %>
           <%= user.suspended? ? "</del>".html_safe : "" %>
+          <br><%= user.email %>
```
Comment (fofr) [inline app/views/admin/users/index.html.erb]: I like these tweaks. User email could probably be set back a bit, and the name emboldened:

screen shot 2015-01-06 at 18 45 19
screen shot 2015-01-06 at 18 45 39

**Comment by fofr** [comment]
Organisation admins can only see users within their organisation. 

In this case:
* it could be helpful to default to that organisation’s filter and show some form of readonly label
* or the filter should be hidden

Perhaps:
screen shot 2015-01-06 at 18 55 04

**Comment by danielroseman** [comment]
All comments addressed except changing to autocomplete.

**Comment by fofr** [comment]
Your changes look good to me. I am working on adding the select2 organisation drop down now.

Closing while I work on it.

**Comment by fofr** [comment]
Have pushed work on a filterable list of organisation in the user filter UI. Re-opening PR.

filtering-organisations

**Comment by danielroseman** [comment]
Changes look good.
Rebased and force pushed.

**Pull request merged**